ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/generator/ChainBuilder.java

```diff
@@ -600,7 +600,11 @@ private SignedBlockAndState generateBlockWithBlobsSidecar(
                 options.getSkipStateTransition()));
 
     final BlobsSidecarSchema blobsSidecarSchema =
-        spec.getGenesisSchemaDefinitions().toVersionEip4844().orElseThrow().getBlobsSidecarSchema();
+        spec.atSlot(slot)
+            .getSchemaDefinitions()
+            .toVersionEip4844()
+            .orElseThrow()
```

Comment:
this looks a lot like where we'd normally be using a `getXXSchemaRequired` call, rather than toVersion.orElseThrow....